library/core/src/sync/atomic.rs

```diff
@@ -31,15 +23,12 @@
 //! The most common way to share an atomic variable is to put it into an [`Arc`][arc] (an
 //! atomically-reference-counted shared pointer).
 //!
-//! [`Sync`]: ../../marker/trait.Sync.html
 //! [arc]: ../../../std/sync/struct.Arc.html
```

Comment:
should this use `[arc]: std::sync::Arc`

Reply:
This does not work because its a link from core to std (#74481)